detections/network/cisco_secure_firewall___high_eve_threat_confidence.yml

```diff
@@ -0,0 +1,58 @@
+data_source:
+- Cisco Secure Firewall Threat Defense Connection Event
+search: |
+  `cisco_secure_firewall` EventType=ConnectionEvent EVE_ThreatConfidencePct >= 80
+  | stats count min(_time) as firstTime max(_time) as lastTime
+      Values(rule) as rule
+      Values(url) as url
+      by EVE_Process, EVE_ThreatConfidencePct, src_ip, dest, dest_port, transport
+  | `security_content_ctime(firstTime)`
+  | `security_content_ctime(lastTime)`
+  | `cisco_secure_firewall___high_eve_threat_confidence_filter`
+how_to_implement: |
+  This search requires Cisco Secure Firewall Threat Defense Logs, which
+  includes the ConnectionEvent EventType. This search uses an input macro named `cisco_secure_firewall`.
+  We strongly recommend that you specify your environment-specific configurations
+  (index, source, sourcetype, etc.) for Cisco Secure Firewall Threat Defense logs. Replace the macro definition
+  with configurations for your Splunk environment. The search also uses a post-filter
+  macro designed to filter out known false positives.
+  The logs are to be ingested using the Splunk Add-on for Cisco Security Cloud (https://splunkbase.splunk.com/app/7404).
+  The access policy must also enable logging.
+known_false_positives: |
+  Some benign applications may exhibit behaviors that resemble encrypted threat patterns, especially if they use uncommon encryption libraries or custom protocols.
+  Custom-developed or internal tools may trigger high EVE confidence scores depending on how they encrypt data.
+  It is recommended to validate the associated process (`EVE_Process`) and destination context, and correlate with other logs (e.g., endpoint or threat intel) before taking response action.
+references:
+  - https://www.cisco.com/c/en/us/td/docs/security/firepower/741/api/FQE/secure_firewall_estreamer_fqe_guide_740.pdf
+rba:
+  message: High threat confidence ($EVE_ThreatConfidencePct$%) from $EVE_Process$ on $src_ip$"
+  risk_objects:
+    - field: src_ip
+      type: other
+      score: 20
+  threat_objects:
+  - field: EVE_Process
+    type: process_name
+tags:
+  analytic_story:
+    - Cisco Secure Firewall Threat Defense Analytics
+  asset_type: Network
+  security_domain: network
+  mitre_attack_id:
+    - T1041
+    - T1071.001
+    - T1105
+    - T1573.002
+  product:
+    - Splunk Enterprise
+    - Splunk Cloud
```

Comment:
Can we add test data by manipulating the raw event to have `EVE_ThreatConfidencePct >= 80`

Reply:
Done